themeadapter-appcompat/gradle.properties

```diff
@@ -14,6 +14,6 @@
 # limitations under the License.
 #
 
-POM_ARTIFACT_ID=accompanist-appcompat-theme
+POM_ARTIFACT_ID=accompanist-themeadapter-appcompat
```

Comment:
Won't this mess people up upgrading? Do we have to have a duplicate deprecated version?

Reply:
TBH I'm not entirely sure, I'll set up some time for us to chat through the migration path

Reply:
Added back the original artifact (with deprecation/migration) as discussed